### PR TITLE
fix(cm6-graphql): Fix query token used as field name

### PR DESCRIPTION
**Pull request by imolorhe**
Fix query token used as field name

Use @extend instead of @specialize
Fixed tests

### CODE REVIEW

File: packages/graphiql/tsconfig.esm.json [hunk 1]
```diff
@@ -23,6 +23,9 @@
     {
       "path": "../codemirror-graphql"
     },
+    {
```
Comment (acao) [inline packages/graphiql/tsconfig.esm.json:26]: this shouldn't be needed until `graphiql` is using `cm6-graphql`

**Comment by acao** [comment]
@imolorhe awesome work! just needs a rebase for an upsteam fix for your project references woes

File: package.json [hunk 2]
```diff
@@ -29,13 +29,14 @@
     "npm": "please_use_yarn_instead"
   },
   "scripts": {
-    "build": "yarn build:clean && yarn build:packages && yarn build:graphiql-react && yarn build:graphiql-plugin-explorer && yarn build:graphiql-plugin-code-exporter && yarn build:graphiql",
+    "build": "yarn build:clean && yarn build:cm6-graphql && yarn build:packages && yarn build:graphiql-react && yarn build:graphiql-plugin-explorer && yarn build:graphiql-plugin-code-exporter && yarn build:graphiql",
```
[the remark this answers is not in the document]
Comment (imolorhe) [reply package.json:32]: `cm6-graphql` doesn't build with `tsc` though. It uses a custom build tool. Does that change anything?

[the remark this answers is not in the document]
Comment (acao) [reply package.json:32]: it works for now, this all needs an overhaul!  fwiw, `wsrun` package has a flag to run a script (like `build`) automatically in order of dependencies

**Review by acao — APPROVED** [review]

**Pull request merged**